integration-tests/tests/src/tests/sync/flexible.ts

```diff
@@ -122,7 +122,6 @@ async function addSubscriptionAndSync<T>(
 
 describe.skipIf(environment.missingServer, "Flexible sync", function () {
   this.timeout(10000); // 10 secs, to allow for long running sync tests
```

Comment:
Remove this timeout.  Locally defined timeouts always overwrite the TIMEOUT environment variable, even if it's larger.